.github/workflows/itk_dict.txt

```diff
@@ -0,0 +1,997 @@
+Abramowitz
+Abscissae
+Acq
+Aerosp
+Agard
+Aghion
+Ai
+Alsabti
+Armijo
+AssembleFforTimeStep
+AssembleKandM
+Austern
+Bankman
+Beucher
+Beucher's
+BiDxz
+Bigler
+Bioimaging
+Bioinformatics
+Biomedicine
+Blas
+Bo
+Borland
+Bracewell
+Brents
+Burstein
+Calc
+Cardano's
+Chrisochoides
+Chu
+CiDxy
+Clatz
+Clunie
+Clunies
+CommandIterationUpdatev
+Comput
+Conners
+ConvertAnitkImageTovtkImageData
+ConvertRGBvtkImageDataToAnitkImage
+Cortesi's
+Creade
+CreateListOfSamplesWithIDs
+Crofton's
+Cuadra
+Cz
+DDataFrameobject
+Dz
+Dasarathy
+Dij
+Dinstein
+DownsamplerType
+Dufour
+Dxx
+Dyx
+Dyy
+Dyz
+Dzx
+Dzy
+Dzz
+Eberhart
+Eigen
+Elmaghraby
+Englewood
+Eun
+Evol
+FEMExceptionItpackSparseMatrixSbagn
+FEMExceptionItpackSparseMatrixSbsij
+FEMSpatialOPbject
+Fedorov
+Fei
+Fillhole
+Fletch
+Freesurfer
+Frenet
+GEAdw
+Gao
+Gibbsian
+Golby
+Graphviz
+Greenleaf
+Gifti
+GPUSuperclass's
+Guillen
+Haar
+Hackathon
+Haralick
+Haralick's
+Hauke
+Heapsort
+Heibel
+Hideaki
+Hierarchique
+Hilden
+Hiraki
+Hirschberg
+Hostname
+IOregion
+ITKIOMeshGifti
+ITKImageToIplImage
+ITKOptimizersv
+Irfan
+Irix
+Isocenter
+Jacobians
+Jianfeng
+Joly
+Josuttis
+Kalman
+Kanungo
+Ke
+Kishore
+KittlerIllingworth's
+LBFGSBOptimizerHelperv
+LBFGSOptimizerBaseHelperv
+LBFGSOptimizerBasev
+Laurienti
+LeBihan's
+Levenberg
+Lewy
+Liao
+Liu
+LoadGrav
+LoadGravConst
+Lz
+Magnotta
+Malaterre
+Mahalanobian
+Mahalanobis
+Marquardt
+Mastronarde
+Meijster
+Methodv
+Minc
+MincTransformIO
+Minkowski
+Musser
+NVidia
+Negahdaripour
+Newmark
+Nicolai
+Nicolson
+Niftilib
+Nikos
+Nocedal
+Nonunit
+NumbxHdrer
+Octant
+Olivo
+Originv
+Oxy
+Oxyz
+Oxz
+Oy
+Oyz
+Pelleg
+Pgrad
+Pimpl
+PimplGlobals
+Pluggable
+PointSetToPointSetMetricWithIndexv
+Polak
+Poynton's
+Preallocate
+PredType
+Prefetch
+Presaturation
+Prev
+Propag
+Pyarali
+Qa
+Rajagopalan
+RCSfile
+RegistrationMethodv
+Req
+Ribiere
+Rician
+Rightarrow
+Rosenbrock
+SeQuence
+Sehgal
+SetBranchness
+SetMedialness
+SetRidgeness
+Shanhbag
+Shanmugam
+Shinin
+SinglephaseChanAndVeseSparseFieldLevelSetSegmentation
+SingleValuedVnlCostFunctionAdaptorv
+Smolikova
+Sobel
+Soille's
+Sonka
+Spall
```

Comment:
Could this be mis-spelling of `Spell`? Or is it a name?

Reply:
Author name J.C. Spall:
https://github.com/InsightSoftwareConsortium/ITK/blob/82d7060408edd452b57b54bd96ee8e2fa037b474/Modules/Numerics/Optimizers/include/itkSPSAOptimizer.h#L57